hardening/image-builder/main.fmf

```diff
@@ -0,0 +1,127 @@
+summary: Runs ImageBuilder remediation and scan inside VMs
+test: python3 -m lib.runtest ./test.py
+result: custom
+environment+:
+    PYTHONPATH: ../..
+duration: 1h
+enabled: false
+require+:
+  # virt library dependencies
+  - libvirt-daemon
+  - libvirt-daemon-driver-qemu
+  - libvirt-daemon-driver-storage-core
+  - libvirt-daemon-driver-network
+  - firewalld
+  - qemu-kvm
+  - libvirt-client
+  - virt-install
+  - rpm-build
+  - createrepo
+  # osbuild library dependencies
+  - osbuild-composer
+  - composer-cli
+extra-hardware: |
+    keyvalue = HVM=1
+    hostrequire = memory>=3720
+adjust:
+  - when: arch != x86_64
+    enabled: false
+    because: we want to run virtualization on x86_64 only
+  # TODO: RHEL-8+ only because osbuild is not on RHEL-7
+tag:
+  - max1
+  - daily
+
+/anssi_bp28_high:
+    environment+:
+        PROFILE: anssi_bp28_high
+    adjust+:
+      - when: distro == rhel-7
+        enabled: false
+        because: RHEL-7 has anssi_nt28_high instead
+    extra-summary: /CoreOS/scap-security-guide/hardening/image-builder/anssi_bp28_high
```

Comment:
I was considering that, but my reasoning was that we have these blocks for all other hardening tests, and it might be easier to just leave them the same, so we can modify all of them in the same manner when removing RHEL-7.

But I have no issues making it RHEL8+ only for `image-builder` either, no strong opinions here.

Reply:
Fine, leave it there for now so it's same with other hardening tests (applies `/hardening/image-builder/tests.py` as well).

Just don't forget to limit distro in in `adjust`.

Reply:
I believe the is no need to change `adjust`, it should already be set up correctly from other tests (that I copied the `.fmf` from).

Reply:
I meant
```
# TODO: RHEL-8+ only because osbuild is not on RHEL-7
```
few lines above.

Reply:
Ah, right, I guess there isn't anything to prevent it running on RHEL-7. I'll add an `adjust`.